test/src/test/java/hudson/model/UpdateCenterErrorsTest.java

```diff
@@ -0,0 +1,185 @@
+package hudson.model;
+
+import com.gargoylesoftware.htmlunit.Page;
+import com.gargoylesoftware.htmlunit.html.DomNodeList;
+import com.gargoylesoftware.htmlunit.html.HtmlButton;
+import com.gargoylesoftware.htmlunit.html.HtmlElement;
+import com.gargoylesoftware.htmlunit.html.HtmlElementUtil;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import hudson.Messages;
+import hudson.util.HttpResponses;
+import jenkins.model.Jenkins;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.TestExtension;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.xml.sax.SAXException;
+
+import javax.annotation.CheckForNull;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+
+public class UpdateCenterErrorsTest {
+    private static final Logger LOGGER = Logger.getLogger(Jenkins.class.getName());
```

Comment:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(UpdateCenterErrorsTest.class.getName());
```

Perhaps easier to understand (otherwise we could ask why using Jenkins there)